docs/proposals/TEMPLATE.md

```diff
@@ -1,4 +1,5 @@
 [This is a template for CSI-Baremetal's changes proposal. ]
+<!-- If you need to draw a diagram, use https://app.diagrams.net/. Don't forget to save it in xlm format as well for edit option.-->
```

Comment:
This note must be changed too, no need xml.

Reply:
Done